### PR TITLE
refactor API calls for wizard Step 1 Satellite

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/routes/deployment.js [hunk 1]
```diff
@@ -38,28 +38,26 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
...
+        controller.set('model.upstream_consumer_name', org.get('upstream_consumer_name'));
       }
     });
   },
 
   loadDefaultData(model, opt) {
     Ember.RSVP.all([
-      request('/api/v2/settings?search=openshift').then(settings => {
-        model.loadOpenshiftDefaults(settings['results'], opt);
+      request('/fusor/api/v21/settings?search=openshift').then(settings => {
```
Comment (cfchase) [inline app/routes/deployment.js]: Seems like this would only work in dev mode if we don't use window.fusorServer somewher in this request.

Comment (isratrade) [reply app/routes/deployment.js]: @cfchase, yep, you're right. will update

File: app/routes/deployment.js [hunk 2]
```diff
@@ -38,28 +38,26 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
...
   loadDefaultData(model, opt) {
     Ember.RSVP.all([
-      request('/api/v2/settings?search=openshift').then(settings => {
-        model.loadOpenshiftDefaults(settings['results'], opt);
+      request(window.fusorServer + '/fusor/api/v21/settings?search=openshift').then(settings => {
+        model.loadOpenshiftDefaults(settings['settings'], opt);
       }),
-      request('/api/v2/settings?search=cloudforms').then(settings => {
-        model.loadCloudformsDefaults(settings['results'], opt);
+      request(window.fusorServer + '/fusor/v21/settings?search=cloudforms').then(settings => {
```
Comment (isratrade) [inline app/routes/deployment.js:59]: @cfchase, squashed this new change of `window.fusorServer` into previous commit.

**Comment by cfchase** [comment]
ACK

**Pull request merged**